src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+import java.time.LocalDateTime;
+
+public class Deadline extends Task {
+
+    static char sym = '\u270F';
```

Comment:
Name should accurately and clearly explain the char, you can consider calling it PENCIL_SYM